mitodl/salt-ops
# enable infinite-corridor on ci and rc, not prod

#### What are the relevant tickets?
For https://github.com/mitodl/open-discussions/issues/3613 and related work

#### What's this PR do?
This PR sets `ENABLE_INFINITE_CORRIDOR=True` on ci and prod, but False on product.

This env variable controls whether or not the `/infinite` page on open-discussions is served or not.

#### How should this be manually tested?
@abeglova Tested this as part of https://github.com/mitodl/open-discussions/pull/3601  :

> Check new ENABLE_INFINITE_CORRIDOR env variable:
> 
> 1. Visit http://yourname.od.odl.local:8063/infinite ... expect 404 ... *Correction: Actually a blank page... maybe we need a 404. But blank page is what open.mit.edu/woofwoofmeow shows, too.*
> 2. Set ENABLE_INFINITE_CORRIDOR=True in env file and docker compose restart watch
> 3. Visit http://yourname.od.odl.local:8063/infinite ... should work
